pandas/io/formats/style.py

```diff
@@ -2257,3 +2414,59 @@ def pred(part) -> bool:
     else:
         slice_ = [part if pred(part) else [part] for part in slice_]
     return tuple(slice_)
+
+
+def _validate_apply_axis_arg(
```

Comment:
this arg validation was added to remove code duplication with #39930 and it might be also be useful for any other dev